packages/plus/src/components/Navigation/constants.ts

```diff
@@ -0,0 +1,35 @@
+import { keyframes } from '@emotion/react'
+
+export const NAVIGATION_MIN_WIDTH = 220 // in px
```

Comment:
should we add the possibility to customize theses constants ?

Reply:
I fixed this by adding a prop `width` and `onWidthChange` that will allow the dev to save and manipulate the width as they need. There is also a complete example in the story